<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VPrincipal</class>
 <widget class="QMainWindow" name="VPrincipal">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CONCESIONARIOS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Source/KWE.ico</normaloff>Source/KWE.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#VPrincipal{
min-height:270px;
min-width: 320px;
max-height: 270px;
max-width: 320px;
}
#panel_principal{
background-color: rgb(234, 234, 234);
border-radius:8px;
}
[objectName^=&quot;btc_btc&quot;]{
background-color: transparent;
border-radius:none;
width:35px;
height:35px;
}
[objectName^=&quot;btc_btc&quot;]:hover{
background-color: rgb(64, 150, 216);
	color: rgb(0, 0, 0);
	border-radius:4px;
}
[objectName*=&quot;btn_btn&quot;]{
background-color: rgb(64, 150, 216);
	color: rgb(0, 0, 0);
	color: rgb(0, 0, 0);
	border-radius:8px;
	font-size: 15px;
	font-weight: bold;
	min-height: 30px;
	max-width: 250px;
}
[objectName*=&quot;btn_btn&quot;]:hover{
background-color: rgb(69, 163, 235);
	color: rgb(0, 0, 0);
}
#panel_encabezado{
min-height: 35px;	
max-height: 35px;
}
#panel_img_logo{
min-height: 120px;
	max-height: 120px;
}
#imgPrincipalMenu{
    margin-left: 60px;
    margin-right: 60px;
}
#VPrincipal{
background-color: rgb(255, 255, 255);
}
#imgPrincipalMenu{
border-radius:8px;
}</string>
  </property>
  <property name="iconSize">
   <size>
    <width>45</width>
    <height>45</height>
   </size>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="panel_principal">
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="panel_img_logo" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="imgPrincipalMenu">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../icono.qrc">:/Source/logo_analytics.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="panel_btn_sucursales" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_btn_kweste">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>-1</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>KENWORTH DEL ESTE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="panel_encabezado" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>310</width>
           <height>14</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btc_btc_cerrar">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/rutas.svg</normaloff>../../../../../Downloads/rutas.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btc_btc_minimizar">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>28</width>
           <height>28</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../icono.qrc"/>
 </resources>
 <connections/>
</ui>
